tests/test_op.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+
+import pytest
+
+import greenplumpython as gp
+
+
+@pytest.fixture
+def db():
+    db = gp.database(host="localhost", dbname="gpadmin")
+    yield db
+    db.close()
+
+
+def test_op_on_consts(db: gp.Database):
+    regex_match = gp.operator("~", db)
+    # FIXME: Remove the single quotes after implementing the const wrapper
+    result = list(regex_match("'hello'", "h.*o").rename("is_matched").to_table().fetch())
+    assert len(result) == 1 and result[0]["is_matched"]
+
+
+def test_op_index(db: gp.Database):
+    import json
+
+    class Student:
+        def __init__(self, name: str, grade: int, courses: List[str]) -> None:
+            self.name = name
+            self.grade = grade
+            self.courses = courses
+
+    john = Student("john", 9, ["math", "english"])
+    # FIXME: Remove type casting after implementing the const wrapper
+    rows = [(f"'{json.dumps(john.__dict__)}'::jsonb",)]
+    student = gp.values(rows, db=db, column_names=["info"]).save_as("student", temp=True)
+    db.execute("CREATE INDEX student_name ON student USING gin (info)", has_results=False)
```

Comment:
As the name implies, this test case is to ensure that when using operators, the user can scan the table using index.

This explains why we need to add operators in GreenplumPython and when the user would need to use them.

Reply:
> As the name implies, this test case is to ensure that when using operators, the user can scan the table using index.

Wouldn't that be a GPDB's behavior instead of GreenplumPython's?

Reply:
I mean if there is an index of a column, and the user is using `@>` to access the table through GreenplumPython, does she/he have an option to use or not use the index?

Reply:
hmmm, I think the test was intended to be the test case for `create_index()`.

Reply:
talked with @xuebinsu. I am OK with this now.